src/main/scala/ch/epfl/scala/decoder/StackTraceFormatter.scala

```diff
@@ -10,6 +10,27 @@ import tastyquery.Types.*
 import scala.annotation.tailrec
 
 class StackTraceFormatter(using ThrowOrWarn):
+  def format(variable: DecodedVariable): String =
+    val typeAscription = variable.declaredType match
+      case tpe: Type => ": " + format(tpe)
+      case tpe => format(tpe)
+    val test1 = formatOwner(variable)
+    val test2 = formatName(variable)
+    formatMethodSignatures(formatOwner(variable)).dot(formatName(variable)) + typeAscription
```

Comment:
Probably we don't need to format the owner of a variable.

```suggestion
     formatName(variable) + typeAscription
```